### PR TITLE
Forgot to repair kbfs icon location

### PR DESCRIPTION
**Pull request by zanderz**
@maxtaco @oconnor663

### CODE REVIEW

File: go/client/cmd_ctl_stop_windows.go [hunk 1]
```diff
@@ -72,6 +72,6 @@ func (s *CmdCtlStop) doKillKBFS() {
 	} else {
 		// open special "file". Errors not relevant.
 		s.G().Log.Debug("KillKBFS: opening .kbfs_unmount")
-		os.Open(filepath.Join(mountDir, ".kbfs_unmount"))
+		os.Open(filepath.Join(mountDir, "\\.kbfs_unmount"))
```
Comment (oconnor663) [inline go/client/cmd_ctl_stop_windows.go]: Shouldn't `Join` insert the slash here automatically?

**Comment by oconnor663** [comment]
LGTM apart form the join question.

**Comment by zanderz** [comment]
It is a valid path without the backslash, so no. I think it stopped working during my testing because the current directory for that volume had changed via CD at the command line, something I hadn't tested or noticed before. Just a guess. This makes it work again for me

**Comment by oconnor663** [comment]
Oh right, the leading backslash makes it an absolute path, so join will let it overwrite all other path components. Except this is Windows so there's "sorta absolute" with no drive letter and "really absolute" with a drive letter, and this is only sorta absolute?

Are you able to print out the results of join before and after this change and see the difference?

**Comment by zanderz** [comment]
yes I saw debug output that .".kbfs_unmount" was invalid, and more importantly, kbfs didn't go down. After the changes, no invalid error, and kbfs went away as intended. Again, it is weird, I tested .kbfs_unmount a lot and worked for me before. This could explain some install/upgrade fails.

**Comment by zanderz** [comment]
here you can see, after navigating to private in the cmd, it thought that was the current path for the mounted drive:
```
2017-01-12T16:52:08.180720 - [DEBU kbfs fs.go:554] 07e => FS CreateFile [tags:DID=iuSpMcSFk4PAT-dM_eCvnA]
2017-01-12T16:52:08.180720 - [DEBU kbfs fs.go:274] 07f open: []string{"private", ".kbfs_unmount"} [tags:DID=iuSpMcSFk4PAT-dM_eCvnA]
2017-01-12T16:52:08.180720 - [DEBU kbfs folderlist.go:71] 080 FL Lookup []string{".kbfs_unmount"} public=false upper=false [tags:DID=iuSpMcSFk4PAT-dM_eCvnA]
2017-01-12T16:52:08.181722 - [DEBU kbfs folderlist.go:131] 081 FL Lookup continuing -> <nil>,TLF name .kbfs_unmount is in an incorrect format [tags:DID=iuSpMcSFk4PAT-dM_eCvnA]
```

**Pull request merged**